Servers/WindowServer/WSMenuManager.cpp

```diff
@@ -47,6 +48,11 @@ void WSMenuManager::draw()
 
     Painter painter(*window().backing_store());
 
+    if (m_dirty) {
+        m_window->set_rect(menubar_rect);
```

Comment:
This should be done before creating the `Painter`, otherwise we are drawing into the old bitmap still.